django/contrib/admin/sites.py

```diff
@@ -299,7 +299,10 @@ def password_change(self, request, extra_context=None):
         if self.password_change_template is not None:
             defaults['template_name'] = self.password_change_template
         request.current_app = self.name
-        return PasswordChangeView.as_view(**defaults)(request)
+        View = PasswordChangeView
+        if not self.logout_template:
+            View.content_type = global_settings.DEFAULT_CONTENT_TYPE
```

Comment:
It's not safe in general, as there's nothing to reset `content_type` back - really a subclass would be needed here, and in the other places this has been dnoe